src/components/menus/content-contextual-menu.tsx

```diff
@@ -605,8 +605,8 @@ const ContentContextualMenu = (props: ContentContextualMenuProps) => {
                 icon: <FileDownload fontSize="small" />,
             });
         }
-
-        if (allowsExportCase()) {
+        // disabled until fix bug from the back side
+        if (!allowsExportCase()) {
```

Comment:
The condition is not good. Keep the same condition and comment all the code nope ?

Reply:
okey